docs/contact_us.md

```diff
@@ -0,0 +1,19 @@
+# Contact Us
+
+## GitHub
+The preferred point of contact for the Cell Ontology is the GitHub [Issue Tracker](https://github.com/obophenotype/cell-ontology/issues/). It is used commonly for requesting new terms or suggesting changes to existing terms. Feel free to open an issue with any kind of request/question/consideration you might have. 
```

Comment:
To make it even easier, I'd suggest replacing
"Feel free to open an issue"
with
"Feel free to open a new issue"
and hyperlinking "open a new issue" with https://github.com/obophenotype/cell-ontology/issues/new/choose.